### PR TITLE
Issue 16852

### PR DESCRIPTION
**Pull request by erickgonzalez**
This PR is for importing files via CSV that the file is a URL, so if the URL is valid it will download the file and create the content. If there is an issue with the URL it will be shown at the preview screen so the user knows that there are some errors with the import.

### CODE REVIEW

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TempFileAPITest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4616566)

Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4616566)

Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4616566)

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4616566)

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4616566)

**Review by jgambarios — APPROVED** [review]

**Comment by jgambarios** [comment]
Failing test https://storage.googleapis.com/cicd-246518-tests/issue-16852/postgres/reports/html/integrationTest/classes/com.dotmarketing.portlets.folders.business.FolderAPITest.html
Probably not related but could be worth to take a look.

**Pull request merged**